kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -371,7 +371,7 @@ object KyuubiConf {
 
   object FrontendProtocols extends Enumeration {
     type FrontendProtocol = Value
-    val THRIFT_BINARY, THRIFT_HTTP, REST, MYSQL, TRINO = Value
+    val THRIFT_BINARY, THRIFT_BINARY_SSL, THRIFT_HTTP, REST, MYSQL, TRINO = Value
```

Comment:
THRIFT_BINARY, THRIFT_BINARY_SSL seems not discrete mathematically

Reply:
I think the idea here is:

1. THRIFT_BINARY, THRIFT_BINARY_SSL should be like HTTP, HTTPS eventually, there is discrete and can be enabled at the same time.
2. currently we keep SSL option of THRIFT_BINARY for compatibility, the deprecating message has been added and this feature will be removed in the 2.0